File: if_else__for__while/if_else__for__while.cpp
#include<iostream>
#include<string>
#include<chrono>
#include<thread>

const std::string x = "1001010011";

int main(){

// if , else if , else
   if (x.size() > 10 ) 
      std::cout << "x.size() > 10" << std::endl;

   else if (x.size() > 20)
      std::cout << "x.size()  is " << x.size() << std::endl;

   else
      std::cout << "x.size() is 10 or less" << std::endl;

// for 
   int j = 0;
   for ( j ; j < 10 ; ++j){
      std::cout << j  << " ";
   }
 

// while 
int i = 0;

while (i < x.size()){
   std::cout << x[i];
   ++i;
   std::chrono::system_clock::time_point  start = std::chrono::system_clock::now();
   
   std::this_thread::sleep_for(std::chrono::seconds(1));  // not good for safety critical coding 
   
   std::chrono::system_clock::time_point finish = std::chrono::system_clock::now();
   std::chrono::duration<double, std::milli>  diff = finish-start; 
   std::cout << "_elapsed time: " << diff.count() << "    ";
   }
  
}

File: if_else__for__while/if_else__for__while.py

# if , elif, else
import time


x = '1000101110'

if len(x) > 10 :
   print("len(x) > 10")
elif len(x) > 20 :
   print ("len(x) is ", len(x))
else :
   print("len(x) is 10 or less")  

# while

i = 0

while i < len(x):
   print(x[i], end="")
   i+=1
   start = time.time()
   time.sleep(1)   #it will not be exactly 1 sec.
   finish = time.time()
   print ( finish - start)
